packages/containers/psammead-timestamp-container/src/timestampUtilities.js

```diff
@@ -30,19 +30,23 @@ export const isValidDateTime = dateTime => {
   return !isNaN(new Date(dateTime)); // eslint-disable-line no-restricted-globals
 };
 
-// when using this function, we recommend using webpack configuration to only load in the relevant timezone, rather than all of moment-timezone
+// when using the following 2 functions, we recommend using webpack configuration to only load in the relevant timezone, rather than all of moment-timezone
+export const unixTimestampToMoment = (timestamp, locale) => {
+  moment.locale(locale);
+  return moment(timestamp);
+};
```

Comment:
I find this a little miss-leading. Instead of only converting unix timestamp to moment, it also sets the locale.

Reply:
The reason it sets the locale here is because the moment needs the current locale of the service to figure out if it needs an alternative calendar [here].(https://github.com/bbc/psammead/blob/530e70f7ec06cadefaa2161eda99b7f17fd461f6/packages/containers/psammead-timestamp-container/src/index.jsx#L44)

Reply:
Ok, But `moment(timestamp)` does not depend on the `locale` to be set, therefore should be moved outside the function.

Reply:
I don't think we need a function just for setting the locale though. How about changing the name to `unixToLocaleMoment`?

Reply:
We don't need another function to set locale. Why do we need to set `locale` though? I can see we are already setting the locale here https://github.com/bbc/psammead/pull/2303/files#diff-9167111d8797dcd3c0e7444464a1d1c7R31-R33. Is there something am missing?

Reply:
Ahh completely missed that you're right. I'll remove the unnecessary locale setter thanks 😃

Reply:
Good spot @sadickisaac but I'd say the lines you link to are problematic, as they set the default moment locale 'globally' (https://momentjs.com/docs/#/i18n/changing-locale/) which can have unwanted side-effects on code that makes assumption about the default locale. Instead we should normally set the locale 'locally' on moment objects we create as described at https://momentjs.com/docs/#/i18n/instance-locale/